maths/bbp_formula.py

```diff
@@ -0,0 +1,105 @@
+def bbp_extract_pi_hex_digit(digit_position: int, precision: int = 1000) -> str:
```

Comment:
Please rename the file to `bailey_borwein_plouffe.py` and rename this function to `bailey_borwein_plouffe()`.